truffle/src/main/java/org/truffleruby/options/ByteStringOptionDescription.java

```diff
@@ -41,7 +41,11 @@ public Object checkValue(Object value) {
 
     @Override
     public String toString(Object value) {
-        return new String((byte[]) value, StandardCharsets.US_ASCII);
+        if (value == null) {
+            return "null";
+        } else {
+            return new String((byte[]) value, StandardCharsets.US_ASCII);
```

Comment:
This is used for the inline script, which is typically input from the the terminal and therefore in the locale encoding aka `Charset.defaultCharset()`.